students/jayjohnson/lesson02/Grid_Printer.py

```diff
@@ -0,0 +1,151 @@
+'''
+Jay Johnson Grid Printer
+
+Write a function that draws a grid like the following:
+
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+
+
+
+'''
+import math
+
+# fuction to print grid
+def print_grid_simple_function(n):
+    print('+ ' + '- ' * n  + '+ ' + '- ' * n  + '+')
+    print('|' + '  ' * n + ' |'  + '  ' * n + ' |')
+    print('|' + '  ' * n + ' |'  + '  ' * n + ' |')
+    print('|' + '  ' * n + ' |'  + '  ' * n + ' |')
+    print('|' + '  ' * n + ' |'  + '  ' * n + ' |')
+    print('+ ' + '- ' * n  + '+ ' + '- ' * n  + '+')
+    print('|' + '  ' * n + ' |'  + '  ' * n + ' |')
+    print('|' + '  ' * n + ' |'  + '  ' * n + ' |')
+    print('|' + '  ' * n + ' |'  + '  ' * n + ' |')
```

Comment:
This works. It's not DRY, so you'd want to look into loops to get this looking less repetitive, but alright!